code/datums/gamemodes/distress.dm

```diff
@@ -423,7 +423,7 @@
 						/obj/item/ammo_magazine/revolver = round(scale * 20),
 						/obj/item/ammo_magazine/smg/m39 = round(scale * 30),
 						/obj/item/ammo_magazine/rifle = round(scale * 25),
-						/obj/item/ammo_magazine/standard_dmrmag = round(scale * 25),
+						/obj/item/ammo_magazine/rifle/standard_dmrmag = round(scale * 25),
```

Comment:
Could you change the name to `/obj/item/ammo_magazine/rifle/standard_dmr` instead? The mag part is already in the name.

Reply:
Alright then